### PR TITLE
build(client-examples): Node16 for multiview-coordinate-interface

### PR DESCRIPTION
**Pull request by jason-ha**
Simple switch to Node16 module as already ESM only and no curated API or tests

### CODE REVIEW

File: examples/data-objects/multiview/interface/package.json [hunk 1]
```diff
@@ -11,13 +11,13 @@
...
 	"author": "Microsoft and contributors",
+	"type": "module",
 	"main": "lib/index.js",
-	"module": "lib/index.js",
 	"types": "lib/index.d.ts",
 	"scripts": {
 		"build": "fluid-build . --task build",
 		"build:compile": "fluid-build . --task compile",
-		"build:esnext": "tsc",
+		"build:esnext": "tsc --project ./tsconfig.json",
```
Comment (DLehenbauer) [inline examples/data-objects/multiview/interface/package.json:20]: @jason-ha - I'm not sure when the '--project ./tsconfig.json' is necessary, but added it for consistency.

File: examples/data-objects/multiview/interface/tsconfig.json [hunk 2]
```diff
@@ -1,9 +1,8 @@
 {
-	"extends": "@fluidframework/build-common/ts-common-config.json",
+	"extends": "../../../../common/build/build-common/tsconfig.node16.json",
 	"compilerOptions": {
-		"module": "esnext",
-		"outDir": "lib",
-		"jsx": "react",
```
Comment (DLehenbauer) [inline examples/data-objects/multiview/interface/tsconfig.json:6]: I removed 'jsx: react' because it's inherited from tsconfig.base.json.

**Review by DLehenbauer — APPROVED** [review]
@jason-ha - Have a look at the two small changes I pushed before merging.

File: examples/data-objects/multiview/interface/package.json (hunk 1 above)
Comment (jason-ha) [reply examples/data-objects/multiview/interface/package.json:20]: `./tsconfig.json` is the default project. I had added it in a number of multi-build cases to make it easier to see which was in use.

**Pull request merged**